core/orchestrator_core.go

```diff
@@ -4487,6 +5380,17 @@ func (o *TridentOrchestrator) updateBackendOnPersistentStore(
 	return nil
 }
 
+// updateVolumeOnPersistentStore updates the volume information in persistent store
```

Comment:
There's some inconsistency in how it generates this.  Sometimes it generates "in **the** persistent store" and sometimes it leaves off the article "**the**" and just generates "persistent store".